src/trafficcontroller/internal/proxy/log_access_middleware.go

```diff
@@ -20,6 +21,12 @@ func NewLogAccessMiddleware(authorize auth.LogAccessAuthorizer) *LogAccessMiddle
 func (m *LogAccessMiddleware) Wrap(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		appID := mux.Vars(r)["appID"]
+		guid := regexp.MustCompile("^[[:xdigit:]]{8}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{12}$")
```

Comment:
I would move this to outside the `HandlerFunc` so we only compile the regexp once at startup and not on each request.